utils/_context/_scenarios.py

```diff
@@ -197,11 +197,10 @@ def _get_warmups(self):
         for container in self._required_containers:
             warmups.append(container.start)
 
-        warmups += [
-            self.agent_container.start,
-            self.weblog_container.start,
-            EndToEndScenario._wait_for_app_readiness,
-        ]
+        warmups += [self.agent_container.start, self.weblog_container.start]
+        # TODO: Support Agent ingestion path in OTel tests
+        if self.weblog_container.library != "java_otel":
+            warmups += [EndToEndScenario._wait_for_app_readiness]
```

Comment:
Why can't the OTel instrumented application expose the endpoint that all apps expose to denote their readiness? It is better to be consistent at the orchestration layer of these processes rather than having many custom things.